packages/esm-patient-banner-app/src/print-identifier-sticker/patient-detail.component.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { age, type CoreTranslationKey, getCoreTranslation, getPatientName, useConfig } from '@openmrs/esm-framework';
+import dayjs from 'dayjs';
+import { type ConfigObject } from '../config-schema';
+import { useTranslation } from 'react-i18next';
+import styles from './print-identifier-sticker.scss';
+
+export interface PatientDetailProps {
+  patient: fhir.Patient;
+}
+
+export const PatientName: React.FC<PatientDetailProps> = ({ patient }) => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <span>
+        <strong className={styles.strong}>{t('patientNameWithSeparator', 'Patient name:')}</strong>
```

Comment:
Hey, right, that's why I have added them in the translation string, rather than harcoding this. This gives the user a choice of their separator and allows change for RTL format.
Thanks!